fbu/PyFBU.py

```diff
@@ -18,7 +18,10 @@ def __init__(self,data=[],response=[],background={},
         #                                     [MCMC parameters]
         self.nTune = 1000
         self.nMCMC = 10000 # N of sampling points
-        self.target_accept = 0.95
+        self.nCores = 1 # number of CPU threads to utilize
+        self.nChains = 2 # number of Markov chains to sample
+        self.nuts_kwargs = None
```

Comment:
I would then suggest to add a flag like: useNUTS_default
and if true, will use the default arguments

Reply:
Is that extra variable necessary though ? The default are used this way.
If one keeps the `nuts_kwargs = None`, this is the default value for the argument in `sample` method, and thus defaults are automatically used according to PyMC3 docs.
If we specify a dictionary with some parameters changed, those parameters will use our settings and other unspecified NUTS parameters will still keep their default values.
But perhaps this behaviour should be described, at least with an in-line comment in the source ?

Reply:
maybe you're right, will then need to add example configs in the notebooks